python/drned_xmnr/op/parse_log_errors.py

```diff
@@ -0,0 +1,71 @@
+def gather_problems(file):
```

Comment:
Hey, this would be a no-brainer for a unit test - no mocking and other nasty stuff needed!

Reply:
definitely! i will add new task for covering this action into backlog.